docs/book/src/tasks/healthcheck.md

```diff
@@ -16,15 +16,16 @@ at the bottom of this page for full details of MachineHealthCheck limitations.
 
 ## What is a MachineHealthCheck?
 
-A MachineHealthCheck is a resource within the Cluster API which allows users to define conditions under which Machines within a Cluster should be considered unhealthy.
+A MachineHealthCheck is a resource within the Cluster API which allows users to define conditions under which Machines within a Cluster should be considered unhealthy. 
```

Comment:
Those changes seems from a commit not relevant for this PR

Reply:
Yeah, since the operator is in a separate branch, these commits come from a rebase. I can remove them from the PR.

Reply:
Unnecessary commits removed